File: aspnet-in-practice/15/VB/VirtualPathProvider/App_Code/AppInitializer.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Web
Imports System.Web.Hosting
Imports ASPNET4InPractice

Public Class AppInitializer
    Public Shared Sub AppInitialize()
        HostingEnvironment.RegisterVirtualPathProvider(New DatabasePathProvider())
    End Sub
End Class

File: aspnet-in-practice/15/CS/VirtualPathProvider/App_Code/AppInitializer.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Hosting;
using ASPNET4InPractice;

public class AppInitializer
{
	public static void AppInitialize()
	{
		HostingEnvironment.RegisterVirtualPathProvider(new DatabasePathProvider());
	}
}